src/Symfony/Component/Console/Command/Command.php

```diff
@@ -605,6 +605,10 @@ public function getUsages()
      */
     public function getHelper($name)
     {
+        if (null === $this->helperSet) {
+            throw new LogicException(sprintf('Unable to retrieve helper "%s" because there is no HelperSet defined for this command. Did you forget to call %s#setHelperSet()?', $name, get_class($this)));
```

Comment:
If I'm right, in Symfony we usually use the `Class::method` syntax instead of `Class#method`. So `%s#setHelperSet()` should be `%s::setHelperSet()` in this error message.